src/main/java/seedu/address/model/person/Person.java

```diff
@@ -35,6 +35,7 @@
     private ObjectProperty<UniqueTagList> tags;
     private ObjectProperty<UniqueRelationshipList> relationships;
 
+    private int sortOption;
```

Comment:
It seems really strange that each person would know how they would be sorted - what does it mean to have a single person have a sort order, especially when not in the context of the list?